audit_exceptions/versioned_dependencies_conflicts_allowlist.json

```diff
@@ -1,5 +1,6 @@
 [
   "anjuta",
+  "hive",
```

Comment:
I think this needs to be in alphabetical order? ...though I suspect tap_syntax will complain about this shortly.

Reply:
Thanks, not sure how that happened (facepalm)

Reply:
No, unfortunately, `tap_syntax` won't complain. It should, though...